lib/railway_ipc/rabbit_mq/rabbit_mq_adapter.ex

```diff
@@ -97,6 +97,7 @@ defmodule RailwayIpc.RabbitMQ.RabbitMQAdapter do
   end
 
   def publish(channel, exchange, payload) do
+    maybe_create_exchange(channel, exchange)
```

Comment:
I'm not totally sure why this maybe function is being used here. It looks like the function pattern matches on if an exchange name as passed in or not. Is there every a case we would call publish without an exchange name?
Also, would we want this to fail out here if an exchange cannot be created? Maybe with:
```suggestion
    :ok = maybe_create_exchange(channel, exchange)
```

Reply:
Yes, we can publish directly to a queue in RPC calls. This code is a bit more coupled than it should be.